<ui version="4.0" >
 <author>Tomeu Borras Riera</author>
 <class>TrabajadorBase</class>
 <widget class="QWidget" name="TrabajadorBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Trabajadores</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../bulmages.qrc" >:/Bulmafact48x48/images/png/48x48/i_bulmages_trabajadores.48x48.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="1" >
    <widget class="QGroupBox" name="groupBox1" >
     <property name="title" >
      <string>Informacion del trabajador:</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>5</number>
      </property>
      <property name="spacing" >
       <number>5</number>
      </property>
      <item row="10" column="0" colspan="3" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>447</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="0" colspan="3" >
       <widget class="QFrame" name="m_frameplugin" >
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3" >
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Numero de la Seguridad Social:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_nsstrabajador" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="1" colspan="2" >
       <widget class="QLineEdit" name="m_emailtrabajador" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>E-mail:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Direccion:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2" >
       <widget class="QLineEdit" name="m_dirtrabajador" />
      </item>
      <item row="5" column="1" >
       <widget class="QLineEdit" name="m_moviltrabajador" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Telefono movil:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="2" >
       <widget class="QPushButton" name="mui_imagen" >
        <property name="text" >
         <string>Cambiar imagen</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Telefono:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLineEdit" name="m_teltrabajador" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Ignored</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="m_apellidostrabajador" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Apellidos:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="m_nomtrabajador" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Nombre:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="m_activotrabajador" >
        <property name="text" >
         <string>Activo</string>
        </property>
       </widget>
      </item>
      <item rowspan="5" row="0" column="2" >
       <widget class="QLabel" name="m_imagen" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <property name="pixmap" >
         <pixmap/>
        </property>
        <property name="scaledContents" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="minimumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../bulmages.qrc" >:/Bulmafact48x48/images/png/48x48/i_bulmages_trabajadores.48x48.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>5</number>
        </property>
        <property name="spacing" >
         <number>5</number>
        </property>
        <item row="0" column="3" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>291</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2" >
         <widget class="QToolButton" name="mui_borrar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Borrar trabajador</string>
          </property>
          <property name="whatsThis" >
           <string>Borrar trabajador</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_editDelete.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QToolButton" name="mui_guardar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Guardar trabajador</string>
          </property>
          <property name="whatsThis" >
           <string>Guardar trabajador</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_save.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QToolButton" name="mui_nuevo" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Nuevo trabajador</string>
          </property>
          <property name="whatsThis" >
           <string>Nuevo trabajador</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../bulmages.qrc" >:/Genericos32x32/images/png/i_fileNew.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title" >
      <string>Lista de trabajadores:</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QListWidget" name="mui_lista" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Box</enum>
        </property>
        <property name="lineWidth" >
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="2" />
 <tabstops>
  <tabstop>mui_nuevo</tabstop>
  <tabstop>mui_guardar</tabstop>
  <tabstop>mui_borrar</tabstop>
  <tabstop>m_activotrabajador</tabstop>
  <tabstop>m_nomtrabajador</tabstop>
  <tabstop>m_apellidostrabajador</tabstop>
  <tabstop>m_dirtrabajador</tabstop>
  <tabstop>m_teltrabajador</tabstop>
  <tabstop>m_moviltrabajador</tabstop>
  <tabstop>m_emailtrabajador</tabstop>
  <tabstop>m_nsstrabajador</tabstop>
  <tabstop>mui_imagen</tabstop>
  <tabstop>mui_lista</tabstop>
 </tabstops>
 <resources>
  <include location="../../bulmages.qrc" />
 </resources>
 <connections/>
</ui>
